app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt

```diff
@@ -320,11 +325,11 @@ object TelemetryWrapper {
     fun stopSession() {
         TelemetryHolder.get().recordSessionEnd()
 
-        if (numLoads > 0) {
-            TelemetryEvent.create(Category.ACTION, Method.FOREGROUND, Object.BROWSER)
-                    .extra(Extra.AVERAGE, averageTime.toString()).queue()
-            resetAverageLoad()
-        }
+        TelemetryEvent.create(Category.ACTION, Method.FOREGROUND, Object.BROWSER)
+                .extra(Extra.HISTOGRAM, Arrays.toString(histogram)).queue()
+
+        // Clear histogram after queueing it
+        histogram = IntArray(HISTOGRAM_SIZE)
```

Comment:
Can you verify with the telemetry team that it's no problem for them to re-use the same event for the histogram? We would have events with the old and new extras format in the database and maybe it would be easier for them to just look at a different event that only contains the new histogram extras.

Reply:
Talked to Frank and he told me to send the keys + values as a JSON String and approved the output.